JenkinsfileStageTrigger

```diff
@@ -0,0 +1,234 @@
+                }
+            }
+        }
+        stage("rsync image build") {
+            when {
+                // Only build if we have related files in changeset
+                changeset "config/Dockerfiles/rsync/**"
+            }
+            steps {
+                script {
+                    tagMap['rsync'] = pipelineUtils.buildImage(openshiftProject, "rsync")
+                }
+            }
+        }
+        stage("ostree-compose image build") {
+            when {
+                // Only build if we have related files in changeset
+                changeset "config/Dockerfiles/ostree_compose/**"
+            }
+            steps {
+                script {
+                    tagMap['ostree-compose'] = pipelineUtils.buildImage(openshiftProject, "ostree-compose")
+                }
+            }
+        }
+        stage("ostree-image-compose image build") {
+            when {
+                // Only build if we have related files in changeset
+                changeset "config/Dockerfiles/ostree-image-compose/**"
+            }
+            steps {
+                script {
+                    tagMap['ostree-image-compose'] = pipelineUtils.buildImage(openshiftProject, "ostree-image-compose")
+                }
+            }
+        }
+        stage("singlehost-test image build") {
+            when {
+                // Only build if we have related files in changeset
+                changeset "config/Dockerfiles/singlehost-test/**"
+            }
+            steps {
+                script {
+                    tagMap['singlehost-test'] = pipelineUtils.buildImage(openshiftProject, "singlehost-test")
+                }
+            }
+        }
+        stage("ostree-boot-image image build") {
+            when {
+                // Only build if we have related files in changeset
+                changeset "config/Dockerfiles/ostree-boot-image/**"
+            }
+            steps {
+                script {
+                    tagMap['ostree-boot-image'] = pipelineUtils.buildImage(openshiftProject, "ostree-boot-image")
+                }
+            }
+        }
+        stage("linchpin-libvirt image build") {
+            when {
+                // Only build if we have related files in changeset
+                changeset "config/Dockerfiles/linchpin-libvirt/**"
+            }
+            steps {
+                script {
+                    tagMap['linchpin-libvirt'] = pipelineUtils.buildImage(openshiftProject, "linchpin-libvirt")
+                }
+            }
+        }
+        stage("Image Tag Report") {
+            steps {
+                script {
+                    // Use tags derived from above image builds
+                    //
+                    pipelineUtils.printLabelMap(tagMap)
+                }
+            }
+        }
+        stage("run stage job") {
+            steps {
+                sleep 30
+                build job: 'upstream-fedora-stage-pipeline',
```

Comment:
Like in the ci-pipeline job, we should run each major release and each have their own stage job